### DISCUSSION TITLE
Side navigation

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#713] [RELEASED: Components, pages and patterns]
The [side navigation component](https://design-patterns.service.justice.gov.uk/components/side-navigation/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by Whersi** [comment]
## Ticket number

https://github.com/ministryofjustice/moj-frontend/issues/159

The details of the ticket is as follows:

**NB: This is a breaking change**

On small screens, [the Sub navigation component](https://moj-design-system.herokuapp.com/components/sub-navigation) changes from a horizontal list into a vertical one. [The side navigation component](https://moj-design-system.herokuapp.com/components/side-navigation) does the opposite, changing from a vertical list into a horizontal one.

A recent accessibility audit highlighted that this hid some of the options by default, effectively making them unavailable for users using magnifiers or larger font sizes.

The small screen view for this component should instead stack the navigation elements as inline blocks, so that they are all visible. This approach is consistent with the Primary navigation and Header components.

Slightly contrived current example (you would need to scroll the component right to see "Nav item 3"):
<img width="204" alt="image" src="https://user-images.githubusercontent.com/100852/120823241-2643d500-c54f-11eb-8e42-12ebc674471f.png">

Intended behaviour:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/100852/120823449-5ab79100-c54f-11eb-8f0d-e4734aa8b181.png">
